oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+
+All, non-deprecated, components of the Open edX project must have a
+named maintainer.  When no such maintainer exists, this will be
+considered an urgent problem to solve.
+
+This OEP proposes a formal definition of the role of maintainer that
+will open to all qualified Open edX community members.  Maintainers
+will be responsible for the health of platform components, but not for
+their performance in production [#]_.
+
+Motivation
+==========
+
+With the change in stewardship from edX to The Center for Reimagining
+Learner, it is essential to have a well defined plan for maintaining
+the platform.
+
+In order to ensure that the project is sustainable for the long term,
+maintenance must be shared by the entire community.  The project must
+be robust enough that a it can tolerate firms and contributors leaving
+the ecosystem.
+
+As a project we must acknowledge that maintenance is hard work, but
+work that must be done for the platform to be fit for use.  The best
+way of ensuring this is to have a named person or persons who are
+responsible for getting the work done.
+
+Specification
+=============
+
+Definitions
+-----------
+
+The term **repository** will mean a repository on the GitHub social
+coding platform.
+
+The term **component** will typically mean the unit of software housed
+in a single GitHub repository.  Exceptions will be made in the case of
+edx-platform where there may be multiple distinguishable components in
+that single repository.  Components will typically, but not always, be
+hosted at the Open edX GitHub organization.  Components will be "part
+of" or "critical to" the open-source platform.
+
+The term **maintainer** will mean a named person responsible for the
+health of component and the component's community.  The remainder of
+this specification will be focused on the details of the role.
+
+The term **core contributor** will mean a named person participating
+in the `Core Contributor Program`_ and meeting the requirements of
+that program.
+
+The term **maintainer scrum-of-scrums** will mean group of all
+maintainers who are responsible for flagging issues to project
+leadership and proposing process improvements for component
+maintenance.
+
+The term **Technical Oversight Committee** (TOC) will mean the group
+responsible for the overall oversight and stewardship of the Open edX
+project. The TOC itself was established and is governed by a formal
+charter.  You can read the full document `here
+<https://openedx.org/wp-content/uploads/2021/12/TCRIL-Technical-Oversight-Committee-Charter-rev.11-16.21.pdf>`_.
+
+Context
+-------
+Historically, the Open edX project has not had a well-defined role
+of maintainer.  At times, community members stepped into this role
+informally.  The role was also part of a broader "owner" role at edX.
+Owners were also responsible for the health of the component they
+owned both in the repository and in production; owners answered the
+pager when something went wrong.
```

Comment:
Again I think I'd love to avoid implicit references to edX or edx.org. This could be more specific by saying "the health of the component they owned both in the repository and in the edx.org production environment; owners answered the pager when something went wrong in that environment"

Reply:
As much as I like the idea of getting to a purely symmetric community, references to edX will help people understand the trajectory toward that ideal.

Reply:
I think this OEP has to do the job of creating clarity for 2U staff as well.  I think this reference is important for historical purposes, to acknowledge that the work was being done be someone, and to explain how it will get done going forward.

Reply:
OK